bin/makisu/cmd/diff.go

```diff
@@ -95,10 +96,23 @@ func (cmd *diffCmd) Diff(imagesFullName []string) error {
 			}
 		}
 		memFSArr = append(memFSArr, memfs)
+
+		// Check image config.
+		imageConfig, err := client.PullImageConfig(manifest.GetConfigDigest())
+		if err != nil {
+			panic(err)
+		}
+		imageConfigs = append(imageConfigs, imageConfig)
+	}
+
+	log.Infof("* Diff two images config")
+	if os.SameFile(imageConfigs[0], imageConfigs[1]) {
+		log.Infof("Image config files are same")
+	} else {
+		log.Infof("Image config files are different")
```

Comment:
Print content in this case.

Reply:
ok.

Reply:
after second look. As for the config, we would just have `fileInfo` which doesn't tell much. we don't have sth like `memFS` to untar the corresponding tar balls to a file system. Should we just tells some basic information of config like `size, name and etc`.